contrib/fonts-tex-gyre-otf/template.py

```diff
@@ -0,0 +1,15 @@
+pkgname = "fonts-tex-gyre-otf"
+pkgver = "2.501"
+_pkgver = pkgver.replace(".", "_")
```

Comment:
as there is only one instance of this being used, just put the .replace with original pkgver directly in the source string